extension/ref-inject-code-search.js

```diff
@@ -41,45 +52,65 @@ var _RESOURCE_MAP = {
-  'android.drm.mobile1'                  : { project:'platform/frameworks/base',     tree:'media/java' },
-  'android.renderscript'                 : { project:'platform/frameworks/base',     tree:'graphics/java' },
-  'android.graphics'                     : { project:'platform/frameworks/base',     tree:'graphics/java' },
-  'android.icu'                          : { project:'platform/frameworks/base',     tree:'icu4j/java' },
-  'android.security'                     : { project:'platform/frameworks/base',     tree:'keystore/java' },
-  'android.location'                     : { project:'platform/frameworks/base',     tree:'location/java' },
-  'android.media'                        : { project:'platform/frameworks/base',     tree:'media/java' },
-  'android.mtp'                          : { project:'platform/frameworks/base',     tree:'media/java' },
-  'android.opengl'                       : { project:'platform/frameworks/base',     tree:'opengl/java' },
-  'android.sax'                          : { project:'platform/frameworks/base',     tree:'sax/java' },
-  'android.telephony'                    : { project:'platform/frameworks/base',     tree:'telephony/java' },
-  'android.net.rtp'                      : { project:'platform/frameworks/base',     tree:'voip/java' },
-  'android.net.sip'                      : { project:'platform/frameworks/base',     tree:'voip/java' },
-  'android.net.wifi'                     : { project:'platform/frameworks/base',     tree:'wifi/java' },
-  'android.support.v4'                   : { project:'platform/frameworks/support',  tree:'v4/java' },
-  'android.support.v7'                   : { project:'platform/frameworks/support',  tree:'v7/appcompat/src' },
-  'android.support.v7.media'             : { project:'platform/frameworks/support',  tree:'v7/mediarouter/src' },
-  'android.support.v7.graphics'          : { project:'platform/frameworks/support',  tree:'v7/palette/src' },
-  'android.support.v13'                  : { project:'platform/frameworks/support',  tree:'v13/java' },
-  'android.support.v17.leanback'         : { project:'platform/frameworks/support',  tree:'v17/leanback/src' },
-  'android.support.design'               : { project:'platform/frameworks/support',  tree:'design/src' },
-  'android.support.customtabs'           : { project:'platform/frameworks/support',  tree:'customtabs/src' },
-  'android.support.percent'              : { project:'platform/frameworks/support',  tree:'percent/src' },
-  'android.support.app.recommendation'   : { project:'platform/frameworks/support',  tree:'recommendation/src' },
-  'android.support.v7.preference'        : { project:'platform/frameworks/support',  tree:'v7/preference/src' },
-  'android.support.v14.preference'       : { project:'platform/frameworks/support',  tree:'v14/preference/src' },
+  'java'                                 : { project:'platform/libcore',                      tree:'ojluni/src/main/java' },
```

Comment:
sorry i think you misunderstood what i asked here... it's hard to review the specific changes given the tabs all changed. can you delete 9 spaces before `tree` so that in GitHub you can more clearly see which lines are unchanged?